### DISCUSSION TITLE
How to add a custom font-family in the editor?

### DISCUSSION THREAD

**Discussion by biomathcode** [outline/rich-markdown-editor#367] [Q&A] [answered]
I am making a journaling app with custom font options. It will be a great help if you can guide me on how can I have an option for adding custom font options. I am decided on this editor. 
How can I increase the line-height vertical and horizontal distance?

**Comment by tommoor** [comment, marked as answer]
I don't think this is a good editor choice for that usecase, the output is stored as Markdown which has no facility to indicate font changes.

**Reply by chumbalayaa** [reply]
Could we not just apply the font changes via CSS after the markdown is rendered in the editor?

The way I'm making changes to the style of the editor is with some simple

`
.ProseMirror {
    min-height: 75vh !important;
}
`
